lib/keila_web/router.ex

```diff
@@ -156,6 +156,12 @@ defmodule KeilaWeb.Router do
     get "/c/:encoded_url/:recipient_id/:link_id/:hmac", TrackingController, :track_click
   end
 
+  scope "/uploads", KeilaWeb do
+    pipe_through :browser
+
+    get "/:filename", LocalFileController, :serve
```

Comment:
Maybe on the future we could make this generic like `FileController` and send the file to Local storage or Remote storage if needed.

Reply:
That was originally my idea, but I think generally it’s probably preferable to use the generated URLs for including files in emails rather than serving them through Keila.